validator/src/bin/solana-test-validator.rs

```diff
@@ -246,7 +246,7 @@ fn main() {
             Arg::with_name("no_bpf_jit")
                 .long("no-bpf-jit")
                 .takes_value(false)
-                .help("Disable the just-in-time compiler and instead use the interpreter for SBF. Windows always disables JIT."),
+                .help("Disable the just-in-time compiler and instead use the interpreter for BPF. Windows always disables JIT."),
```

Comment:
Oops can you please restore this line

Reply:
Fixed at ae6f93f303bd9c48db48b9488276c9a0006097c8